frame/plasm-lockdrop/src/lib.rs

```diff
@@ -518,12 +601,16 @@ impl<T: Trait> Module<T> {
 
     fn btc_issue_amount(value: u128, duration: T::Moment) -> u128 {
         // https://medium.com/stake-technologies/plasm-lockdrop-introduction-99fa2dfc37c0
+        // Alpha = a * 1000
```

Comment:
It is recommended that you use PerThings for accurate decimal point calculations.
https://crates.parity.io/sp_arithmetic/trait.PerThing.html

Reply:
Oh, yes, it's a good point!

Reply:
Fixed @satellitex